Vue/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
For the Vue framework, let's move the existing ```.eslintrc.js``` config to the [our eslint config repository](https://github.com/DevExpress/eslint-config-devextreme) in ```vue.js``` config file.

We don't override any rules in this config (only extends), but it helps us use the same eslint config across all of our Vue applications :)